androidbrowserhelper/src/androidTest/AndroidManifest.xml

```diff
@@ -83,6 +105,12 @@
             <meta-data
                 android:name="android.support.customtabs.trusted.STATUS_BAR_COLOR"
                 android:resource="@color/status_bar_color" />
+
+            <intent-filter>
```

Comment:
We should probably put a comment somewhere that this is required for the shortcuts to work.